subsys/fb/cfbv_1016.c

```diff
@@ -0,0 +1,1256 @@
+/*
+ * This file was automatically generated using the following command:
+ * /home/john/Software/zephyrproject/zephyr/scripts/build/gen_cfb_font_header.py -i DroidSansMono.ttf -x 10 -y 16 -s 14 --center-x --y-offset -2 -o /home/john/Software/zephyrproject/zephyr/subsys/fb/cfbv_1016.c
```

Comment:
Should I remove this? the original font file, whilst being generated still had the copyright notice for both Phytec (who checked in the file I think) and google (who made the font).
The file as it is right now has not been tampered with, it's exactly what the script spits out.